src/Constants_Conversion.f90

```diff
@@ -30,24 +30,20 @@ MODULE Constants_Conversion
   IMPLICIT NONE
   PRIVATE
 
-  REAL(SRK),PUBLIC,PARAMETER ::   ZERO=0.000000000000000_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::    ONE=1.000000000000000_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::    TWO=2.000000000000000_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::  QTRPI=7.853981633974483E-1_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER :: HALFPI=1.570796326794897E+0_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::     PI=3.141592653589793E+0_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::  TWOPI=6.283185307179586E+0_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER :: FOURPI=1.256637061435917E+1_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::    ZERO=0.000000000000000_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::     ONE=1.000000000000000_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::     TWO=2.000000000000000_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::   QTRPI=7.853981633974483E-1_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::  HALFPI=1.570796326794897E+0_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::      PI=3.141592653589793E+0_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::   TWOPI=6.283185307179586E+0_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::  FOURPI=1.256637061435917E+1_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::     RPI=1.0_SRK/PI
+  REAL(SRK),PUBLIC,PARAMETER ::  RTWOPI=1.0_SRK/TWOPI
+  REAL(SRK),PUBLIC,PARAMETER :: RFOURPI=1.0_SRK/FOURPI
   !> Avogadro's Number
   REAL(SRK),PUBLIC,PARAMETER :: Na=6.02214129000E+23_SRK !16 digits
 
-  !> Direction enumeration
-  !INTEGER(SIK),PARAMETER :: WEST=1
-  !INTEGER(SIK),PARAMETER :: NORTH=2
-  !INTEGER(SIK),PARAMETER :: EAST=3
-  !INTEGER(SIK),PARAMETER :: SOUTH=4
-  !INTEGER(SIK),PARAMETER :: BOTTOM=5
-  !INTEGER(SIK),PARAMETER :: TOP=6
   !> Conversion factor from MPa to psia (should move to constants conversion)
```

Comment:
These should only be defined in SurfaceIndexing.f90 in ```MPACT_libsReactor``` unless there is another use for them in Futility.

Reply:
Yeah, Brendan's right.  There's no need for the surface indexing in Futility.  These ones were commented out because we had already moved them to the SurfaceIndexing module.  They should have been deleted at that point instead of being commented.

Reply:
I was actually referring to the comment that says "(should move to constants conversion)" in the line right below

Reply:
Oh... I didn't even notice the comment.  Oh well.